app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  validates :username, unique: true, presence: true
+  validates :email, presence: true
```

Comment:
It is unique in the migration, but we could add it here as well. No harm in having it in both places.